build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -93,6 +71,11 @@ public static string GetSdkDebianPackageName(BuildTargetContext c)
             return $"{packagePrefix}-dev-{nugetVersion}";
         }
 
+        public static string GetDebianHostFxrPackageName(BuildTargetContext c)
```

Comment:
It should be called. I just updated it. The CI failed because of not calling it.